core/src/stored/bcopy.cc

```diff
@@ -173,6 +171,11 @@ int main(int argc, char* argv[])
   my_config = InitSdConfig(configFile.c_str(), M_CONFIG_ERROR);
   ParseSdConfig(configFile.c_str(), M_CONFIG_ERROR);
 
+  if (input_archive.empty() || output_archive.empty()) {
+    printf(T_("%sNothing done."), AvailableDevicesListing().c_str());
```

Comment:
`Nothing done.` will not get printed as `stdout` is linebuffered by default.
```suggestion
    printf(T_("%sNothing done.\n"), AvailableDevicesListing().c_str());
```

I would also like to have the output to actually include what the problem was.
Currently the only feedback i get is
```console
$ ./sbin/bls-restore -c etc/bareos -V TestVolume001
Available Devices:
 "FileStorage" (storage)
```
which doesnt tell me what to do.  It should say something like
```
$ ./sbin/bls-restore -c etc/bareos -V TestVolume001
Missing input device. Available Devices:
 "FileStorage" (storage)
```